spec/controllers/collection_items_controller_spec.rb

```diff
@@ -15,7 +30,66 @@
       end
     end
 
-    context "create" do
+    context "rejected parameter for collection with items in" do
+      let(:owner) { @collection.owners.first.user }
+      render_views
```

Comment:
Rather than testing the views, can we simply test that the value of the `@collection_items` instance variable is being correctly assigned in the controller? Like:
```
      it "includes rejected items" do
        fake_login_known_user(owner)
        get :index, collection_id: @collection.name, rejected: true
        expect(assigns(:collection_items)).to include @rejected_work_item
      end
```
Checking the page for the collection title and the work title feels a little like we're wandering away from testing the code that's actually in the controller.